web/server/entrypoint.go

```diff
@@ -57,6 +57,22 @@ type robotServer struct {
 	registry *logging.Registry
 }
 
+func logViamEnvVariables(logger logging.Logger) {
+	var viamEnvVariables []interface{}
```

Comment:
Let's also grab the home directory via `PlatformHomeDir` [here](https://github.com/viamrobotics/rdk/blob/37df2537fc9cc7c2eae92c39e73156fae3544339/utils/env.go?plain=1#L60).

(and looking at that function -- it should be using Go's stdlib `os.UserHomeDir()` because maybe windows one day)

Reply:
added